stories/components/daily/BolusTooltip.js

```diff
@@ -0,0 +1,479 @@
+import React from 'react';
+
+import { storiesOf } from '@kadira/storybook';
+
+import BolusTooltip from '../../../src/components/daily/bolustooltip/BolusTooltip';
+
+// bolus' lifted from bolus utils tests
```

Comment:
Just wondering what your thoughts are on whether or not it's better to move this into a shared data fixture.  I'm honestly not sure which I prefer.  I like code reuse, but it's also nice to have the context in place when it's just being used in tests/stories.

Reply:
Yeah, I had considered moving it into a shared fixture, but the value in being able to just scroll to/jump to the definition in the same file outweighed the DRY aspect for me, but that may have just been because of how often I was referencing it during development. If we find ourselves using the fixture anywhere else, I'd strongly recommend collapsing it.